app/components/Table.js

```diff
@@ -124,7 +147,7 @@ function Table({
                       },
                     ])}
                   >
-                    {cell.render("Cell")}
+                    <CellWrapper>{cell.render("Cell")}</CellWrapper>
```

Comment:
The additional padding here causes the content to be misaligned with the headers:

![image](https://user-images.githubusercontent.com/380914/129486052-68cf8a17-9237-4247-9032-fd7663a513e2.png)

I think the aim of improved spacing on mobile is good though, my suggestion would be:

- Apply padding to both the table `td` / `th` elements directly
- Remove the padding on first-child/last-child so that the left and right edges still align